packages/sdk-react/src/core/componentSettings/index.ts

```
@@ -21,7 +21,26 @@ import type { MoniteIconWrapperProps } from '@/ui/iconWrapper';
 import type { I18n } from '@lingui/core';
 import { t } from '@lingui/macro';
 
-interface ReceivableSettings extends MoniteReceivablesTableProps {
+import {
+  defaultAvailableCountries,
```

Biome (2.1.2)

[error] Shouldn't redeclare 'defaultAvailableCountries'. Consider to delete it or rename it.

'defaultAvailableCountries' is defined here:


(lint/suspicious/noRedeclare)